src/Tizen.Diagnostics/Tizen.Diagnostics/Diagnostics.cs

```diff
@@ -0,0 +1,298 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using NativeApi = Interop.Diagnostics;
+
+namespace Tizen.Diagnostics
+{
+    /// <summary>
+    /// The Diagnostics class is used for requesting client data and receiving diagnostics report events.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public static class Diagnostics
+    {
+        private static NativeApi.NotificationCallback _notificationCallback;
+        private static event EventHandler<ReportReceivedEventArgs> _reportReceived;
+
+        /// <summary>
+        /// Event to be invoked when new report context arrives.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public static event EventHandler<ReportReceivedEventArgs> ReportReceived
+        {
+            add
+            {
+                if (_reportReceived == null)
+                    SetNotificationCallback();
+                _reportReceived += value;
+            }
+            remove
+            {
+                _reportReceived -= value;
+                if (_reportReceived == null)
+                    UnsetNotificationCallback();
+            }
+        }
+
+        /// <summary>
+        /// Requests other apps to dump diagnostics data.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <param name="clientId">An id of app or service to request.</param>
+        /// <param name="parameters">Array of parameters passed to the client.</param>
+        /// <returns>
+        /// The instance of DiagnosticsData.
+        /// </returns>
+        /// <privilege>
```

Comment:
Empty documentation tag.

Reply:
According to the comment: https://code.sec.samsung.net/jira/browse/ACR-1581?focusedCommentId=1137103&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1137103
It's prohibited to mention internal privilege in public API, so Mr. @kd0228-kim decided to leave it blank and add remarks tag with proper info.

Reply:
I agree that "internal" privilege should not be included in the "public" API reference, but leaving an empty documentation tag is also not acceptable, because it means nothing for the API user (it won't be even displayed by the IDE, I am not sure about docFX later). Comment of @kd0228-kim applies for the native API documentation, but in C# API it would be better to remove this tag entirely and leave the additional info in the `remarks` tag (as you have already done).

Reply:
Ok, done.